examples/teleport-usage/README.md

```diff
@@ -0,0 +1,38 @@
+# Teleport Usage Gathering Script
+
+This script retrieves the number of unique users accessing each of the five Teleport supported protocols.
```

Comment:
This says 5 support protocols, but the script below only supports 4 (no kube yet)

Reply:
This usage tool does calculate usage for Kube access. The kube issue I outlined in the readme is that when you run the script in Kubernetes the TTY behavior does not allow for user input as it requires you press enter before starting and it errors out because it assumes an empty response.